apps/desks.py

```diff
@@ -294,8 +294,12 @@ def __send_notification(self, updates, desk):
                     desk=desk.get("name"),
                 )
                 push_notification("activity", _dest=activity["recipients"])
+                get_resource_service("users").update_stage_visibility_for_user(user)
```

Comment:
I see `get_resource_service("users")` 4 times in this function, might be good to store the service locally use it thereafter.